src/moj/components/multi-file-upload/multi-file-upload.js

```diff
@@ -199,7 +199,7 @@ if (
       url: this.params.deleteUrl,
       type: 'post',
       dataType: 'json',
-      data: data,
+      data,
```

Comment:
Very much a personal preference but I don't like this style at all! It's not saving any typing, and to me it feels less obvious what is happening, especially when it is mixed in with other `key: value` pairs. But I accept that part of this whole process is removing personal preference infavour of consistent/enforced code style.